File: Front-end/BusinessLayer/Models/SpaceDTO.js
export default class SpaceDTO {
  constructor(hasId) {
    if (hasId) {
      this.id = "";
    }
    this.name = "";
    this.size = 0;
    this.volume = 0;
    this.description = "";
    this.bookings = [];
    this.spaceConsumables = [];
  }
}

File: Back-end/BusinessLayer/Models/SpaceDTO.cs
namespace BusinessLayer.Models;

public class SpaceDTO
{
    public Guid Id { get; set; }

    public string? Name { get; set; }

    public double Size { get; set; }

    public double Volume { get; set; }

    public string? Description { get; set; }

    public virtual ICollection<BookingDetailsDTO>? Bookings { get; set; } = new List<BookingDetailsDTO>();

    public virtual ICollection<SpaceConsumableDTO>? SpaceConsumables { get; set; } = new List<SpaceConsumableDTO>();
}